documentation/smg/02_development_environment.md

```diff
@@ -1 +1,215 @@
 # Development Environment
+
+## Scope
+
+This document describes setting up a Horace/Herbert environment for new starters at STFC/ISIS.
+
+This is version 1 by Chris Marooney.
+At the time of writing, my available hardware was a personal laptop;
+an STFC laptop will not be available for some weeks.
+This affects my choice of environment.
+
+## Dependencies
+
+The following system requirements are detailed individually below as they are used,
+but are summarized here for convenience:
+- cmake (3.7 or greater)
+- Visual Studio (2017 or greater;for Windows)
+- g++ (for Linux)
+- Matlab (R2018b or R2019b,from the STFC-specific page.)
+
+## Environments
+
+Available environments are:
+
+- Personal laptop with Windows 10 working from home.
+- IDaaS (ISIS Data Analysis as a Service) VM running linux.
+
+The following steps may vary according to your environment;
+please select appropriately.
+
+## Getting Horace and Herbert
+
+Horace lives in two separate code-bases.
```

Comment:
I know what you're saying here, but the phrasing is slightly confused: Horace lives in one repository and has a runtime dependence on Herbert, which lives in a separate repository.

Frustratingly Horace is a runtime dependency for some pieces of Herbert at the moment too but there are open tickets to try to sort that.

Reply:
rephrased to be more in line with your text. You may wish to comment further (when pushed).